ecommerce/models.py

```diff
@@ -45,3 +48,15 @@ class Line(Model):
 
     created_at = DateTimeField(auto_now_add=True)
     modified_at = DateTimeField(auto_now=True)
+
+
+class CoursePrice(Model):
+    """
+    Information about a course run's price and other ecommerce info
+    """
+    course_run = ForeignKey(CourseRun)
+    price = DecimalField(decimal_places=2, max_digits=20)
+    is_valid = BooleanField(default=False)
+
+    created_at = DateTimeField(auto_now_add=True)
+    modified_at = DateTimeField(auto_now=True)
```

Comment:
I'm not sure that kind of constraint exists on the database side, but I can write an API function to enforce this